domain/adapter.go

```diff
@@ -1,6 +1,7 @@
 package domain
 
 type AdapterResult struct {
+	Output string
 	Error  error
```

Comment:
We should also replace this error wrapping here. But let's do this in a separate PR 😄